pyreal/transformers/scale.py

```diff
@@ -0,0 +1,124 @@
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
+from sklearn.preprocessing import Normalizer as SklearnNormalizer
+from sklearn.preprocessing import StandardScaler as SklearnStandardScaler
+
+from pyreal.transformers import Transformer
+from pyreal.transformers.wrappers import DataFrameWrapper
+
+
+class MinMaxScaler:
+    """
+    basically a sklearn MinMaxScaler but maintains DataFrame type
+    """
+
+    def __init__(self, feature_range=(0, 1), *, copy=True, clip=False):
+        """initialize a wrapped transformer and DataFrameWrapper, then wrap the DataFrameWrapper
+
+        Args:
+            feature_range (tuple (min, max), default=(0, 1)):
+                Desired range of transformed data.
+            copy (bool, default=True):
```

Comment:
The copy and feature_range parameters both do not exist in the base Transformer class; would they both end up in the base Transformer class? If this is the case I'm concerned about the feature_range parameter interfering with the base Transformer class (because I can't get rid of it; it doesn't have a default value).